.github/workflows/avail-listing-validator.py

```diff
@@ -0,0 +1,93 @@
+import json
+import requests
+import sys
+
+JSON_URL = "https://raw.githubusercontent.com/karnotxyz/avail-campaign-listing/main/listing.json"
```

Comment:
shouldn't this be the listing.json file of the PR

Reply:
This is the main branch JSON URL, Needed to check duplication in `check_duplicate_urls_in_latest_entry`